package.json

```diff
@@ -27,7 +27,7 @@
     "prepare": "npm run build",
     "prepublishOnly": "npm test && npm run lint",
     "graphql:codegen": "graphql-codegen --config graphql-codegen.yml",
-    "swagger:codegen": " openapi --input https://raw.githubusercontent.com/cowprotocol/services/v2.227.1/crates/orderbook/openapi.yml --output src/order-book/generated --exportServices false --exportCore false",
+    "swagger:codegen": " openapi --input https://raw.githubusercontent.com/cowprotocol/services/v2.245.1/crates/orderbook/openapi.yml --output src/order-book/generated --exportServices false --exportCore false",
```

Comment:
why not getting from latest from master the openapi?

Reply:
Is that what you want? I don't mind, I'm only following the convention that was here previously.

Reply:
I'm almost sure I suggested to use release rather than master to have a better control/tracking of where the current version comes from.
If pointing to a branch, it's harder to tell which commit was used as a reference.

If we don't want to use a release, should we use a commit on `main` branch instead?
This way the update is intentional and clear.

Reply:
I'm fine with point to a version, I actually think is safer and agree with it being more reproducible. 
My only concern is, we won't remember to update from time to time, if this doesn't bother us on commit. 

One alternative is to make a check if we use the latest version.